### DISCUSSION TITLE
How to run two instances of frigate on same set of cameras?

### DISCUSSION THREAD

**Discussion by PovilasID** [blakeblackshear/frigate#5899] [General]
Hey,

I run two instances of frigate: 
LOCAL - server in the house.
REMOTE - server in the cloud.

I have VPN running that gives access to REMOTE server to all my cameras and LOCAL server.
In REMOTE server's config should I point to RTSP streams of `go2rtc` streams or to cameras directly?

**Comment by blakeblackshear** [comment]
Why wouldn't you just access the local server remotely over the VPN? Or backup the local server?

**Reply by PovilasID** [reply]
**VPN access is not enough** I want to have remote copy of video because it is unattended remote property and even if a burglar gets detected it will take a while for anybody to get to it. Long enough that taking the local server with all the footage of the burglary is likely.

**Backups are too slow.** They run on schedule and it comes back to robber leaving with the footage.

**Connectivity is too flaky remote network drive** being a primary storage for clips. 

I will look into how to use something like rsync to push files to remote as soon as files are written and the connection is up... 
However, I was previously testing frigate on a remote server and I had an instance already running, soo I just pointed the streams over VPN to and it worked! Well, sort of... probably because of flaky connectivity remote server has only a few detections. Though connectivity may not be entirely at fault, hence the question. 

Also, I prefer having the same UI for viewing footage. Because of it is likely I will keep remote version running in parallel even if I turn on a storage sync option. 
Ideally local server would 'federate' detections to remote with option push over the footage.

P.S. Connectivity is flaky but bandwidth is ok... so as long it is on it can handle a couple of extra streams

**Comment by blakeblackshear** [comment]
I would probably point the remote server at the cameras directly to eliminate a point of failure if the cameras can handle it.

**Reply by PovilasID** [reply]
Did some testing and when using go2rtc streams for remote server it was not stable when trying to watch in webrtc mode, so I did switch to the cam. For now looks OK. 
I will do some detection testing later this week.
